pkg/catalog/catalog.go

```diff
@@ -157,7 +157,19 @@ func getRegistryDevfiles(registry Registry) (registryDevfiles []DevfileComponent
 
 		err = json.Unmarshal(jsonBytes, &devfileIndex)
 		if err != nil {
-			return nil, errors.Wrapf(err, "Unable to unmarshal the devfile index.json from %s", indexLink)
+			if err := util.CleanDefaultHTTPCacheDir(); err != nil {
+				log.Warning("Error while cleaning up cache dir.")
+			}
+			// we try once again
+			jsonBytes, err := util.HTTPGetRequest(request, cfg.GetRegistryCacheTime())
+			if err != nil {
+				return nil, errors.Wrapf(err, "Unable to download the devfile index.json from %s", indexLink)
```

Comment:
the error message should start with a small letter

Reply:
done